src/sync/Backoff.zig

```diff
@@ -0,0 +1,29 @@
+const std = @import("std");
+const builtin = @import("builtin");
+const Backoff = @This();
```

Comment:
can we change this to use the struct { } approach we use throughout the codebase - were not following zig's approach - would be nice to also add this note to the contributing.md - will also want to change the file to lowercase (backoff.zig)

Reply:
Got it!